v2/cli/init_command.go

```diff
@@ -187,7 +188,8 @@ func addToGitignore(pathToIgnore string, gitIgnorePath string) error {
 	return nil
 }
 
-// Given a language, check if it is an allowed language and canonicalize it
+// canonicalizeLanguage checks if a given lagnuage is allowed. If it is,
+// shorten it to its file extension. Otherwise, return an error.
 func canonicalizeLanguage(language string) (string, error) {
 	switch language {
```

Comment:
Should probably convert the input to lower case first. As it stands now, there's an unwritten assumption that the caller has.